packages/renderer/src/lib/preferences/PreferencesConnectionActions.svelte

```diff
@@ -153,9 +153,9 @@ function getLoggerHandler(
 
 {#if connectionStatus}
   {#if connection.lifecycleMethods && connection.lifecycleMethods.length > 0}
-    <div class="mt-2 relative">
+    <div class="mt-2 relative" role="group" aria-label="Connection Actions">
```

Comment:
it's not clear why we need to move some group definition at an upper level

Reply:
So they would be on the same level.

Reply:
From testing point of view it does not matter after all. will revert.